UnitTests/Tests/TemperatureSpec.swift

```diff
@@ -0,0 +1,64 @@
+import Foundation
+import Quick
+import Nimble
+
+class Temperaturespec: QuickSpec {
+    override func spec() {
+        describe("Temperature") {
+            
+            func compare(temp1: Int, temp2: Int) -> TemperatureComparison {
+                let t1 = Temperature(fahrenheitValue: temp1)
+                let t2 = Temperature(fahrenheitValue: temp2)
+                return t1.comparedTo(t2)
```

Comment:
Might be more useful when we have more in swift than obj-c but what about implementing `Comparable` on `Temperature` so we can `t1 < t2`?